test_utils/src/cluster.rs

```diff
@@ -34,13 +45,25 @@ impl Cluster {
         info!("###### Creating new cluster ######");
         info!("Validator keys:");
         let k = keys(None);
+        let mut nodes = HashMap::new();
 
         for (index, key) in k.into_iter().enumerate() {
             info!("Key {index} -> {}", key.public().clone());
+
+            nodes.insert(
+                index,
+                Arc::new(NodeDetails {
+                    id: index,
+                    name: key.public().clone(),
+                    registry: Registry::new(),
+                    store_path: Default::default(),
```

Comment:
Since this is a `String`, isn't there a risk that if you then call `start_node` with `preserve_store`, this will lead to a start at the system root (`/`) ? 

One way to get out of this conundrum, I think is to use [`env::temp_dir`](https://doc.rust-lang.org/std/env/fn.temp_dir.html) - i.e. on a Unixoid the equivalent of the $TMPDIR env variable - for the default value of the path. In many cases, this will resolve to `/tmp`. This is not `tempdir` from the associated crate, which on an Unixoid resolves to an `mktemp`.

Reply:
Ah very good catch, I'll fix this